code/game/objects/items/storage/boxes.dm

```diff
@@ -297,35 +287,35 @@
 		/obj/item/light_bulb/tube/large,
 		/obj/item/light_bulb/bulb,
 	)
-
-/obj/item/storage/box/lights/mixed/Initialize(mapload, ...)
-	. = ..()
-	for(var/i in 1 to 14)
-		new /obj/item/light_bulb/tube/large(src)
-	for(var/i in 1 to 7)
-		new /obj/item/light_bulb/bulb(src)
+	spawns_with = list(
+		/obj/item/light_bulb/tube/large, /obj/item/light_bulb/tube/large, /obj/item/light_bulb/tube/large,
+		/obj/item/light_bulb/tube/large, /obj/item/light_bulb/tube/large, /obj/item/light_bulb/tube/large,
+		/obj/item/light_bulb/tube/large, /obj/item/light_bulb/tube/large, /obj/item/light_bulb/tube/large,
+		/obj/item/light_bulb/tube/large, /obj/item/light_bulb/tube/large, /obj/item/light_bulb/tube/large,
+		/obj/item/light_bulb/tube/large, /obj/item/light_bulb/tube/large,
+		/obj/item/light_bulb/bulb, /obj/item/light_bulb/bulb, /obj/item/light_bulb/bulb,
+		/obj/item/light_bulb/bulb, /obj/item/light_bulb/bulb, /obj/item/light_bulb/bulb,
+		/obj/item/light_bulb/bulb,
+	)
```

Comment:
would definitly be better with the other format.
You can use spawn mult to simply too

Reply:
I intended to use spawns_mult for this but I completely forgot